lib/webdrivers/chromedriver.rb

```diff
@@ -115,7 +134,11 @@ def driver_filename(driver_version)
         elsif System.platform == 'linux'
           'linux64'
         elsif System.platform == 'mac'
-          apple_filename(driver_version)
+          if driver_version >= normalize_version('115')
```

Comment:
If we move this conditional to `#apple_filename` we don't need to add `#apple_filename_api`.